rm/rm-client/src/main/java/org/ow2/proactive/resourcemanager/common/event/RMNodeSourceEvent.java

```diff
@@ -67,7 +66,7 @@
     private String nodeSourceName = null;
 
     /** description of the source concerned by the event. */
-    private BigString nodeSourceDescription = null;
+    private String nodeSourceDescription = null;
```

Comment:
NodeSource descriptions can not be passed by clients. In the constructor of NodeSource you can see that the descriptions are created as 

        this.description = "Infrastructure: " + im + ", Policy: " + policy;

This description is never saved on the DB :) but generated on startup or on nodesource creation time and kept in memory.